### PR TITLE
Flake8 Docstrings

### LINKED ISSUE #9: Completing the move to flake8-docstrings
We've decided to add `flake8-docstrings` to project requirements for this repository.
We've agreed on the following list of errors to ignore:
```sh
ignore=
    P102,B311,W503,E226,S311,
    # Missing Docstrings
    D100,D104,D107,
    # Docstring Whitespace
    D202,D203,D204,D212,D214,D215,
    # Docstring Quotes
    D301,D302,
    # Docstring Content
    D401,D402,D405,D406,D407,D408,D409,D410,D411,D412,D413, D414
```

### PR DESCRIPTION
**Pull request by GhostofGoes**
Add flake8 docstring configuration and add docstrings to make code pass linting.

Closes #9

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+ignore=
+    P102,B311,W503,E226,S311,
+    # Missing Docstrings
+    D100,D104,D107,
+    # Docstring Whitespace
+    D202,D203,D204,D212,D214,D215,
+    # Docstring Quotes
+    D301,D302,
+    # Docstring Content
+    D401,D402,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
```
Comment (MarkKoz) [inline .flake8]: The ignores haven't been finalised yet. `D400` has been ignored recently as well https://github.com/python-discord/seasonalbot/pull/146#issuecomment-474549550.

File: snekbox.py [hunk 2]
```diff
@@ -27,13 +29,23 @@ def __init__(self,
         'PYTHONDONTWRITEBYTECODE': '1',
     }
 
-    def nsjail_workaround(self):
+    def _nsjail_workaround(self):
```
Comment (MarkKoz) [inline snekbox.py]: Missing docstring. To be fair I have no idea what this workaround is.

File: .flake8 (hunk 1 above)
Comment (sco1) [reply .flake8]: Add D400 to ignores

File: config.py [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import os
 import docker
 from docker.errors import NotFound
-# import traceback
 
 
 def autodiscover():
+    """Search for the snekbox container and return it's IPv4 address."""
```
Comment (sco1) [inline config.py]:
```suggestion
    """Search for the snekbox container and return its IPv4 address."""
```

File: snekbox.py (hunk 2 above)
Comment (GhostofGoes) [reply snekbox.py]: I made it a private method, which don't require docstrings

Comment (GhostofGoes) [reply snekbox.py]: I didn't know what it did either, and it was only used internally, hence making it private

**Pull request merged**